tests/kvasir-tests/rijndael/Makefile

```diff
@@ -7,10 +7,13 @@ EXTRA_OUTPUTS := cbc_d_m.txt cbc_e_m.txt ecb_d_m.txt ecb_e_m.txt ecb_iv.txt \
 DTRACE_DIFF := dtrace-diff-fast.pl
 COUNTS_ONLY := 1
 
-# This test seems to be non-deterministic on CentOS.
+# This test seems to be non-deterministic on Rocky Linux.
 OS_ID := $(shell grep -oP '(?<=^ID=).+' /etc/os-release | tr -d '"')
```

Comment:
It would be nice to delete this line and move tests after include ../Makefile.common

Reply:
Done, in a different pull request.